proxy/core/src/test/scala/io/cloudstate/proxy/crdt/AbstractCrdtEntitySpec.scala

```diff
@@ -28,6 +28,8 @@ object AbstractCrdtEntitySpec {
       |akka.actor.provider = cluster
       |# Make the tests run faster
       |akka.cluster.distributed-data.notify-subscribers-interval = 50ms
+      |akka.actor.allow-java-serialization = on
+      |akka.actor.warn-about-java-serializer-usage = off
```

Comment:
@sean-walsh Why do we want to silence these warnings?

Reply:
We probably don't. Also, See Heiko's comment above, should I just change the test to use another serializer?